xonsh/completers/xompletions.py

```diff
@@ -0,0 +1,46 @@
+import xonsh.xontribs as xx
```

Comment:
Can you add a module docstring, please? Also can you add a sphinx autodoc hook for this module too, please? Thanks!